File: PHP/uberBlack.php
<?php
    require_once('car.php');
    class UberBlack extends Car{
        $typeCarAccepted = array();
        $seatsMaterial = array();

        public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial){
            parent::__construct($license, $driver)
            $this->typeCarAccepted = $typeCarAccepted;
            $this->seatsMaterial = $seatsMaterial;
        }
    }


    /*$fruits = array (
        "fruits"  => array("a" => "Orange", "b" => "Banana", "c" => "Apple"),
        "numbers" => array(1, 2, 3, 4, 5, 6),
        "holes"   => array("first", 5 => "second", "third")
    ); */


?>

File: Python/uberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, driver, brand, model):
        super.__init__(license,driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial